### DISCUSSION TITLE
kimai in iframe: nothing happens when users submit login credentials

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#4793] [Questions & Answers]
Hello,

We would like to use kimai within an iframe in [Hive Pages](https://help.hive.com/en/articles/6777410-introducing-pages).
Unfortunately, when a user submits his credentials, nothing happens.
When a user tries to log in with Google SAML, the following error message occurs:
![image](https://github.com/kimai/kimai/assets/148647197/77b8574f-2d35-4405-87cb-9af91861edc4)

Using kimai in an iframe on our website works perfectly.

Any suggestions what we could do to get it to work?
Thanks. :-)

**Comment by kevinpapst** [comment]
> We would like to use kimai within an iframe 

Okay

> Unfortunately, when a user submits his credentials, nothing happens.
> When a user tries to log in with Google SAML, the following error message occurs:

So it doesn't work

> Using kimai in an iframe on our website works perfectly.

And now it works. 😕

**Reply by EduNetEurope** [reply]
No, it works on our website.
But it does not work in Hive.

**Reply by kevinpapst** [reply]
Why do you ask here if it does not work in Hive?
